msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Build apps for Teams meeting stage
+author: v-sdhakshina
+description: In this article, learn how to build meeting apps API references that are available for Teams client.
+ms.topic: conceptual
+ms.author: v-sdhakshina
+ms.localizationpriority: medium
+ms.date: 04/07/2022
+---
+
+# Build apps for Teams meeting stage
```

Comment:
Should we provide guidance on using frameContext and `meetingStage` to customising the UI shared to stage. So that partners create Meeting apps that actually are customised to the experience?